src/pages/project/project_wizard/NFRProject.tsx

```diff
@@ -1,3 +1,73 @@
-export const NFRProject = () => {
-  return <p>NFR Project</p>;
+import EastOutlinedIcon from '@mui/icons-material/EastOutlined';
+import PaidOutlinedIcon from '@mui/icons-material/PaidOutlined';
+import { Box, Button } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { NfrProject, ProjectOrganization, SaveCristinProject } from '../../../types/project.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { getLanguageString } from '../../../utils/translation-helpers';
+import { CreateProjectAccordion } from './CreateProjectAccordion';
+import { NfrProjectSearch } from './NfrProjectSearch';
+
+interface NFRProjectProps {
+  newProject: SaveCristinProject;
+  setNewProject: (val: SaveCristinProject) => void;
+  setShowProjectForm: (val: boolean) => void;
+  coordinatingInstitution: ProjectOrganization;
+}
+
+export const NFRProject = ({
+  newProject,
+  setNewProject,
+  setShowProjectForm,
+  coordinatingInstitution,
+}: NFRProjectProps) => {
+  const { t } = useTranslation();
+  const [selectedProject, setSelectedProject] = useState<NfrProject | null>(null);
+
+  const onCreateProject = () => {
+    if (!selectedProject) {
+      return;
+    }
+
+    setNewProject({
+      ...newProject,
+      title: getLanguageString(selectedProject.labels),
+      startDate: selectedProject.activeFrom,
+      endDate: selectedProject.activeTo,
+      coordinatingInstitution,
+      funding: [
+        {
+          type: 'UnconfirmedFunding',
+          source: selectedProject.source,
+          identifier: selectedProject.identifier,
+          labels: selectedProject.labels,
+        },
+      ],
+    });
+    setShowProjectForm(true);
+  };
+
+  return (
+    <CreateProjectAccordion
+      headline={t('project.form.start_with_nfr_financing')}
+      description={t('project.form.start_with_nfr_financing_details')}
+      testId={dataTestId.newProjectPage.createNFRProjectAccordion}
+      icon={<PaidOutlinedIcon sx={{ height: '3rem', width: '3rem', mr: '0.75rem' }} />}>
+      <Box sx={{ display: 'flex', flexDirection: 'column' }}>
```

Comment:
samme som i `EmptyProjectForm`, tenker det hadde vært naturlig at container har gap heller enn at button har margin.
Om ikke container skal ha gap, kunne faktisk hele den `<Box>`en vært tatt vekk, og knappen kunne hatt `placeSelf: 'end'` for å få den til høyre